src/sheets/updateStatus.js

```diff
@@ -0,0 +1,26 @@
+import {updateItem} from './updateItem'
+
+export async function updateStatus(order, items, status) {
+  const addCompanyItemData = (result, sheetName, rowIndex, item) => {
+    result.push({
+      range: `${sheetName}!H${rowIndex}:H${rowIndex}`,
+      values: [[status]],
+    })
+  }
+
+  const addPersonalItemData = (result, sheetName, rowIndex, item) => {
+    result.push({
+      range: `${sheetName}!E${rowIndex}:E${rowIndex}`,
+      values: [[status]],
+    })
+  }
+
+  await updateItem(
+    order.isCompany,
+    items,
+    addCompanyItemData,
+    addPersonalItemData,
+  )
+}
+
```

Comment:
Remove empty lines at the end of the file.

Reply:
Done.